op-chain-ops/genesis/config.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/eth"
 )
 
+// initialzedValue represents the `Initializable` contract value. It should be kept in
+// sync with the constant in `Constants.sol`.
+// https://github.com/ethereum-optimism/optimism/blob/b6104508cb11ac85252484e212c199acf512a874/packages/contracts-bedrock/src/libraries/Constants.sol#L49
+const initialzedValue = 3
```

Comment:
Typo in variable name
```suggestion
// initializedValue represents the `Initializable` contract value. It should be kept in
// sync with the constant in `Constants.sol`.
// https://github.com/ethereum-optimism/optimism/blob/b6104508cb11ac85252484e212c199acf512a874/packages/contracts-bedrock/src/libraries/Constants.sol#L49
const initializedValue = 3
```